client/src/public/ts/opponent.ts

```
@@ -71,45 +74,33 @@ class Opponent {
+    this.statsText.text = `♥︎${health} 🡕${combo} 🞎${stock}`;
+
+    // enemies
+    for (const enemy of data.enemies) {
       this.updateEnemy(enemy.id, data);
     }
-    let enemyToDeleteMatches = data.enemiesToErase;
-    for (let enemyID of enemyToDeleteMatches) {
-      let enemyToDelete = this.stageItems.sprites[`enemy${enemyID}`];
+    const enemyToDeleteMatches = data.enemiesToErase;
+    for (const enemyID of enemyToDeleteMatches) {
+      let enemyToDelete = this.enemies[`enemy${enemyID}`];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'enemyToDelete' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)